ladder/emeritus_members.yaml

```diff
@@ -0,0 +1,2 @@
+members:
+  - nobody
```

Comment:
Please add eneritus members, refer to https://github.com/dragonflyoss/dragonfly/blob/main/OWNERS.md.